deploy/aws/eks/manifests/local-volume-provisioner.yaml

```diff
@@ -36,12 +36,7 @@ spec:
     spec:
       tolerations:
         - key: dedicated
-          operator: Equal
-          value: pd
-          effect: "NoSchedule"
-        - key: dedicated
-          operator: Equal
-          value: tikv
+          operator: Exists
```

Comment:
does this mean it will run on TiDB nodes now?

Reply:
Yes

Reply:
TiDB does not need local SSD disk so that isn't desirable in this setup.

Reply:
Why not leave it how it was?

Reply:
Because the values of taints are updated to ${cluster_name}-tikv in favor of multiple cluster management.

And yes, TiDB nodes don't have to run the provisioner pod, I will add an extra label to address this problem

Reply:
I've addressed this in the latest commit😜